test/fuzz/fuzzing.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+chmod u=rwx ./test/fuzz/fuzzing.sh
```

Comment:
This does not make sense to me. If the script was not executable, then I cannot execute it to make it executable. The permissions are anyway stored in git so it will always get the correct permissions as long as it was committed with them (which it was).
```suggestion
```